test/unit/classification_test.rb

```diff
@@ -560,17 +560,286 @@ def setup
     as_admin do
       LookupValue.create! :lookup_key_id => key.id,
                           :match => "hostgroup=#{parent_hostgroup}",
-                          :value => "parent",
+                          :value => ['parent'],
+                          :omit => false
+      LookupValue.create! :lookup_key_id => key.id,
```

Comment:
@bagasse,
Is it necessary to have Lookup Value with organization match in every test?

Reply:
In some case this not not related specifically to organization, but just create a Lookup Value that is on the Lookup Values "search path" to test when `merge_overrides` is set to true or false we have the good result (merged or not)